include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -653,7 +653,8 @@ def AIE_PacketFlowOp: AIE_Op<"packet_flow", [SingleBlockImplicitTerminator<"EndO
 
   let arguments = (
     ins AIEI8Attr:$ID,
-        OptionalAttr<BoolAttr>:$keep_pkt_header
+        OptionalAttr<BoolAttr>:$keep_pkt_header,
+        OptionalAttr<BoolAttr>:$ctrl_pkt_flow
```

Comment:
What does `ctrl_pkt_flow=true` do? Is it documented?  Does it make it "route before other flows"?

Reply:
Yes exactly, this flag labels packet flow ops which get routed before others, so that the goal is they always get routed to use the same master/slave/arbiter/msel. When control packets reconfigure the control packet flow, things tend to fall apart leading to deadlock.

Let me add documentation.

Reply:
Doc. added.

Reply:
Make sense. For now the ctrl packet flows were only special in that they get routed first. I renamed that attr to "priority_route".